### PR TITLE
Adding OLS bundle installation

### PR DESCRIPTION
**Pull request by JoaoFula**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: .tekton/integration-tests/pipelines/lightspeed-integration-test-pipeline.yaml [hunk 1]
```diff
@@ -61,15 +134,34 @@ spec:
...
+              export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}
+              curl -Lo /usr/local/bin/operator-sdk ${OPERATOR_SDK_DL_URL}/operator-sdk_linux_${ARCH}
+              chmod +x /usr/local/bin/operator-sdk
+              operator-sdk version
               echo "---------------------------------------------"
-              echo ${SNAPSHOT} | jq .
+              oc new-project $(params.namespace)
+              export IMAGE_DIGEST_MIRROR_SET=https://raw.githubusercontent.com/openshift/lightspeed-operator/main/config/manager/imagedigestmirrorset.yaml
+              curl -Lo /usr/tmp/imagedigestmirrorset "$IMAGE_DIGEST_MIRROR_SET"
+              oc apply -f /usr/tmp/imagedigestmirrorset
```
Comment (raptorsun) [inline .tekton/integration-tests/pipelines/lightspeed-integration-test-pipeline.yaml:158]: If we are using the test bundle that is already done with image substitution, we could ignore this `imagedigestmirrorset`

Comment (JoaoFula) [reply .tekton/integration-tests/pipelines/lightspeed-integration-test-pipeline.yaml:158]: yeah, lemme remove it

**Pull request merged**